bin/tessel-2.js

```diff
@@ -99,11 +99,17 @@ makeCommand('run')
   })
   .option('slim', {
     flag: true,
-    help: 'Bundle only the required modules'
+    default: true,
+    help: 'Bundle only the required files'
   })
   .option('slimPath', {
     default: 'build.js'
   })
+  .option('full', {
+    flag: true,
+    default: false,
+    help: 'Bundle all files in project'
```

Comment:
We may want to make this more explicit: "Bundle all files in project including those not used by the program".